src/fheroes2/gui/interface_list.h

```diff
@@ -400,6 +419,7 @@ namespace Interface
                 needRedraw = true;
 
                 _topId = _size() - maxItems;
+                _currentId = std::clamp( _topId + maxItems, _currentId, _size() - 1 );
```

Comment:
By the end button we want to position on the last item in the list so we can simply:
```suggestion
                _currentId = _size() - 1;
```

Reply:
I think somehow I got the Page Down code in the End section. Fixed.